marklogic-data-hub-central/ui/src/components/entities/mapping/mapping-step-detail/mapping-step-detail.test.tsx

```diff
@@ -512,41 +518,74 @@ describe("RTL Source-to-entity map tests", () => {
+    let entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-1");
     validateMappingTableRow(entTable, ["propId", "propName", "propAttribute", "items", "gender"], "name", data.mapProps.entityTypeProperties);
 
     //Click on the Name column to sort the rows by Ascending order
     fireEvent.click(entityTableNameSort);
-    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-0");
+    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-1");
     validateMappingTableRow(entTable, ["gender", "items", "propAttribute", "propId", "propName"], "name", data.mapProps.entityTypeProperties);
 
+    //Entity type title should remain in the first row after sort is applied
+    entTableRow = document.querySelectorAll("#entityContainer .ant-table-row-level-0");
+    entTableRow.forEach(item => { if (item.getAttribute("data-row-key") === "0") { return entTableTopRow = item; } });
+    expect(entTableTopRow).toHaveTextContent(data.mapProps.entityTypeTitle);
+
     //Click on the Name column again to sort the rows by Descending order
     fireEvent.click(entityTableNameSort);
-    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-0");
+    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-1");
     validateMappingTableRow(entTable, ["propName", "propId", "propAttribute", "items", "gender"], "name", data.mapProps.entityTypeProperties);
 
     fireEvent.click(entityTableNameSort); //Reset the sort order to go back to default order
 
     //Click on the Type column to sort the rows by Ascending order
     fireEvent.click(entityTableTypeSort);
-    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-0");
+    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-1");
     validateMappingTableRow(entTable, ["int", "ItemType [ ]", "string", "string", "string"], "type", data.mapProps.entityTypeProperties);
 
     //Click on the Type column again to sort the rows by Descending order
     fireEvent.click(entityTableTypeSort);
-    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-0");
+    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-1");
     validateMappingTableRow(entTable, ["string", "string", "string", "ItemType [ ]", "int"], "type", data.mapProps.entityTypeProperties);
 
     //Resetting the sort order to go back to default order
     fireEvent.click(entityTableTypeSort);
-    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-0");
-    validateMappingTableRow(entTable, ["int", "string", "string", "ItemType [ ]", "string"], "type", data.mapProps.entityTypeProperties);
+    entTable = document.querySelectorAll("#entityContainer .ant-table-row-level-1");
+    // validateMappingTableRow(entTable, ["int", "string", "string", "ItemType [ ]", "string"], "type", data.mapProps.entityTypeProperties);
```

Comment:
Same suggestion here. Let's remove the unused code entirely instead of commenting it, unless it might be required in future.

Reply:
Thanks, removed.